src/reformat_libyuv.c

```diff
@@ -341,6 +341,24 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     return AVIF_RESULT_NOT_IMPLEMENTED;
 }
 
+// These defines are used to to create a NULL reference to libyuv functions that
+// did not exist prior to a particular version of libyuv.
+#if LIBYUV_VERSION < 1838
+#define I422ToRGB565Matrix NULL
```

Comment:
Nit: This code may potentially break if libavif is compiled against a modified version of libyuv. For example, a copy of libyuv may have local modifications that cherry-pick these functions from future versions.

It would be better to define a macro like the following:

```
#if LIBYUV_VERSION >= 1838
#define I422_TO_RGB565_MATRIX I422ToRGB565Matrix
#else
#define I422_TO_RGB565_MATRIX NULL
#endif
```

And then use `I422_TO_RGB565_MATRIX` in the code below.